pkg/actions/store.go

```diff
@@ -25,17 +25,17 @@ type Store interface {
 }
 
 type KVStore struct {
-	store kv.StoreMessage
+	store kv.Store
```

Comment:
Maybe we want to change the type from `KVStore` to `ActionStore`? This might be a bit confusing

Reply:
The package is 'actions' so ActionStore will just repeat the name. The 'KVStore' is the implementation of the 'Store' interface used by the actions package.
Will change it to `kvStore` as it is an implementation of `Store` interface using the kv.